### PR TITLE
card component added

### PR DESCRIPTION
**Pull request by MonikaJhingan**

### CODE REVIEW

File: Card/card.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+.card-discount {
+  font-size: inherit;
+  color: var(--btn-secondary-color);
+}
+
+.btn-shopping-cart {
+  align-self: center;
+  background-color: var(--primary-color);
+  /* background-color: transparent; */
```
Comment (RakshithBhat03) [inline Card/card.css]: You can remove this comment

File: Card/card.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  height: 4rem;
+  width: 18rem;
+  border-radius: 1rem;
+  border: none;
+  font-size: 1rem;
+  cursor: pointer;
+  color: rgb(235, 152, 152);
```
Comment (RakshithBhat03) [inline Card/card.css]: You can use a variable here

File: Card/card.css [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  font-size: 6.9rem;
+  font-weight: bold;
+  z-index: 9999;
+  height: 50rem;
+  width: 35rem;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.text-card {
+  background-color: pink;
```
Comment (RakshithBhat03) [inline Card/card.css]: Same here as well.

File: Card/card.css [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+  box-shadow: 10px 10px 45px -13px rgba(0, 0, 0, 0.75);
+}
+.code-container {
+  width: 100%;
+}
+
+.code-iframe{
+  border:0; 
+  transform: scale(1);
+   overflow:hidden;
```
Comment (RakshithBhat03) [inline Card/card.css]: Code is not indented properly. Use a code formatter.

File: Card/card.css [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+.card-title {
+  height: 3rem;
+  width: 36rem;
+  justify-content: space-between;
+  font-weight: bold;
+}
+
+.wishlist-icon {
+  color: rgb(196, 17, 17);
```
Comment (RakshithBhat03) [inline Card/card.css]: You can use a variable here.

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested a few changes, everything else looks good.

File: Card/card.css (hunk 1 above)
Comment (MonikaJhingan) [reply Card/card.css]: done

File: Card/card.css (hunk 2 above)
Comment (MonikaJhingan) [reply Card/card.css]: done

File: Card/card.css (hunk 3 above)
Comment (MonikaJhingan) [reply Card/card.css]: done

File: Card/card.css (hunk 4 above)
Comment (MonikaJhingan) [reply Card/card.css]: done

File: Card/card.css (hunk 5 above)
Comment (MonikaJhingan) [reply Card/card.css]: done

File: Card/card.css [hunk 6]
```diff
@@ -0,0 +1,151 @@
+@import url("../../Utility CSS/navbar.css");
+
+.box {
+  display: flex;
+  flex-direction: row;
```
Comment (atharvabhanage02) [inline Card/card.css]: There's no need to mention flex-direction : row , it's row by default

**Review by atharvabhanage02 — APPROVED** [review]
There are few blank spaces , remove them as well . 
Else everything looks fine

File: Card/card.css (hunk 6 above)
Comment (MonikaJhingan) [reply Card/card.css]: done

**Review by sharmivijay — APPROVED** [review]
Code looks neat and you can add the colors in the variables files

**Comment by MonikaJhingan** [comment]
> Code looks neat and you can add the colors in the variables files

definitely!!!

**Pull request merged**